x/oracle/types/params.go

```diff
@@ -18,13 +18,17 @@ var (
 	KeySlashFraction            = []byte("SlashFraction")
 	KeySlashWindow              = []byte("SlashWindow")
 	KeyMinValidPerWindow        = []byte("MinValidPerWindow")
+	KeyStampPeriod              = []byte("StampPeriod")
+	KeyPrunePeriod              = []byte("PrunePeriod")
 )
 
 // Default parameter values
 const (
 	DefaultVotePeriod               = BlocksPerMinute / 2 // 30 seconds
 	DefaultSlashWindow              = BlocksPerWeek       // window for a week
 	DefaultRewardDistributionWindow = BlocksPerYear       // window for a year
+	DefaultStampPeriod              = BlocksPerDay        // window for a day
+	DefaultPrunePeriod              = BlocksPerMonth      // window for a month
```

Comment:
What if a block time will change? It can be updated without changing a code, through a gov voting.

Reply:
Then we'll have to change these definitions for the entire oracle module, that's not a part of this PR